manifests/charts/istio-operator/files/gen-operator.yaml

```diff
@@ -173,7 +173,7 @@ spec:
       serviceAccountName: istio-operator
       containers:
         - name: istio-operator
-          image: gcr.io/istio-testing/operator:1.14-dev
+          image: "gcr.io/istio-testing/operator:1.14-dev"
```

Comment:
hmmm, do not think that the double quotes are needed, this change will make this one instance different from others, would not think this is needed.

Reply:
@litong01 I enclosed `image` with double quotes because other occurrences for `image` are double quoted.

Reply:
Majority of the image values are not quoted. the only places that I can see quoted are places where scripts are needed. Other places that use mixed (quoted and unquoted) are from testdata which should not have been. I would prefer not to quote it. If you leave this one and the one above alone, your PR can have one less file changes.